src/components/split-button/split-button.component.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/icon';
+import Button from '../button';
+import StyledSplitButtonContainer, { StyledToggleButton } from './split-button.style';
+import StyledSplitButtonChildrenContainer from './split-button-children.style';
+import { validProps } from '../../utils/ether/ether';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Events from '../../utils/helpers/events';
+
+class SplitButton extends React.Component {
+  static propTypes = {
+    /**
+     * Customizes the appearance, can be set to 'primary' or 'secondary'.
+     */
+    as: PropTypes.string,
```

Comment:
Prop 'as' can be quite problematic - https://www.styled-components.com/docs/api#as-polymorphic-prop

Reply:
Yeah it gets renamed before it’s passed to the styled component, had to leave it for backwards compatibility purposes